### PR TITLE
feat: bump kubernetes-client to 6.4.0 (#172)

### LINKED ISSUE #172: bump to kubernetes-client 6.4.0
related to https://github.com/redhat-developer/intellij-kubernetes/issues/500
depends on https://github.com/fabric8io/kubernetes-client/pull/4689

Kubernetes-client is 6.4.0 should include the fixes that are required for it to work properly on the IJ platform. We should migrate to this version once it's released.

### PR DESCRIPTION
**Pull request by adietish**
fixes #172

### CODE REVIEW

File: src/main/java/com/redhat/devtools/intellij/common/utils/MetadataClutter.java [hunk 1]
```diff
@@ -42,7 +42,6 @@ public class MetadataClutter {
     );
 
     private static final List<Consumer<ObjectMeta>> setters = Arrays.asList(
-            metadata -> metadata.setClusterName(null),
```
[the remark this answers is not in the document]
Comment (jeffmaury) [reply src/main/java/com/redhat/devtools/intellij/common/utils/MetadataClutter.java:45]: That lower what was done. Can't we try to manage this field through additionalProperties ?

[the remark this answers is not in the document]
Comment (adietish) [reply src/main/java/com/redhat/devtools/intellij/common/utils/MetadataClutter.java:45]: @jeffmaury: not a bad idea, looking...

**Review by jeffmaury — APPROVED** [review]
LGTM

**Comment by adietish** [comment]
This will get merged as soon as https://github.com/redhat-developer/intellij-kubernetes/pull/548 is ready to be merged.

File: build.gradle [hunk 2]
```diff
@@ -20,18 +20,19 @@ intellij {
...
     withJavadocJar()
 }
 
 dependencies {
-    compile (
-        'io.fabric8:kubernetes-client:5.12.4',
-        'io.fabric8:openshift-client:5.12.4',
+    implementation (
+        'io.fabric8:kubernetes-client:6.3.0',
+        'io.fabric8:openshift-client:6.3.0',
```
Comment (adietish) [inline build.gradle:34]: should be bumped to 6.4.0 once it's released

**Comment by adietish** [comment]
@jeffmaury, @lstocchi: kubernetes-client 6.4.0 was released. Making this ready to be merged.

**Review by lstocchi — APPROVED** [review]
LGTM

**Comment by adietish** [comment]
@jeffmaury, @lstocchi: confirm LGTM?

**Comment by lstocchi** [comment]
> @jeffmaury, @lstocchi: confirm LGTM?

yes :+1:

**Pull request merged**